CHANGELOG.rst

```diff
@@ -7,6 +7,193 @@ Changelog
     The Dutch version of this changelog can be found :ref:`here <changelog-nl>`.
 
 
+3.2.0 "Zomer" (2025-07-11)
+==========================
+
+Open Forms 3.2.0 is a feature release.
+
+.. epigraph::
+
+    "Zomer" is Dutch for "Summer".
+
+This contains the changes from the alpha releases and fixes applied until the stable version.
+BEFORE upgrading to 3.2.0, please read the release notes carefully and review the following
+instructions.
+
+Upgrade procedure
+-----------------
+
+# TODO-5398: which version?
+⚠️ Before upgrading to the 3.2 release series, please ensure you upgrade to Open Forms x.y.z first.
```

Comment:
I haven't squashed any migrations, so there's no minimum version requirement this time.